cluster/addons/calico-policy-controller/calico-node.yaml

```diff
@@ -0,0 +1,107 @@
+kind: DaemonSet
+apiVersion: extensions/v1beta1
+metadata:
+  name: calico-node
+  namespace: kube-system
+  labels:
+    kubernetes.io/cluster-service: "true"
+    k8s-app: calico-node
+spec:
+  selector:
+    matchLabels:
+      kubernetes.io/cluster-service: "true"
+      k8s-app: calico-node
+  template:
+    metadata:
+      labels:
+        kubernetes.io/cluster-service: "true"
+        k8s-app: calico-node
+      annotations:
+        scheduler.alpha.kubernetes.io/critical-pod: ''
```

Comment:
If this runs in a cluster w/o alpha features enabled, do these annotations get ignored?